frontend/src/LogIn/LogIn.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-layout
+    column
+    justify-center
+    align-center>
```

Comment:
`v-layout` `column` and `v-flex` are from vuetify version 2.
We are using vuetify version 3, you need to use the grid system of that version.
https://vuetifyjs.com/en/components/grids#grid-system